### ISSUE TITLE
Profile login error "please log in again" 

### ISSUE THREAD

**Issue by astrogexo** [gitcoinco/web#8775]
**Describe the bug**
After logging in a banner appears with message "Unable to save your profile. Please log in again". Error does not go away after loggin out and loggin back in.

Can close error banner and navigate to other gitcoin.co pages but error pops up again.

**To Reproduce**
Log in for the first time using github account. 

**Expected behavior**
Can create a profile.

**Screenshots**

![GitCoinProfileError](https://user-images.githubusercontent.com/32209658/114585028-3b883b80-9c51-11eb-9ca5-69a696bde970.JPG)

**Desktop (please complete the following information):**
Tried with both Chrome Version 89.0.4389.114 (Official Build) (64-bit)
and Firefox 87.0
on Windows 10, Version 1909

**Smartphone (please complete the following information):**

Chrome 89.0.4389.105 

for Android 11 on Samsung Galaxy S21.

**Comment by danielzak** [comment]
Same error for me. Had this issue with this Github account for at least 2 years (probably forever).

I have tried to revoke all access rights on the Github side and try again. Does not work. 
I have tried to setup a new Github account and login to Gitcoin. It works fine.

Is the problem srelated to the Github account itself, or some property in it? I would really appreciate this issue solved as I'd like to interact with Gitcoin using my primary Github account.
